include/RAJA/policy/cuda/kernel/CudaKernel.hpp

```diff
@@ -301,8 +301,13 @@ struct CudaLaunchHelper<cuda_launch<async0, num_blocks, num_threads>,StmtList,Da
     }
   }
 
+  //
+  // actual_threads can be modified in the case where
+  // the requested number of theads would cause the
+  // max_blocks to be 0
+  //
   inline static void max_blocks(int shmem_size,
-      int &max_blocks, int actual_threads)
+      int &max_blocks, int &actual_threads)
```

Comment:
actual_threads should not be changed at this point if num_threads was specified at compile-time.